src/mpid/ch4/include/mpidpre.h

```diff
@@ -244,6 +244,7 @@ typedef enum {
 } MPIDIG_win_info_accumulate_ops;
 
 typedef enum {
+    MPIDIG_ACCU_OP_FIRST = 0,
```

Comment:
The naming convention for this enum is super messed up.  For now, can you change this to `MPIDIG_ACCU_OP_SHIFT_FIRST` to match `MPIDIG_ACCU_OP_SHIFT_LAST`.  We can do a separate commit to change everything to `MPIDI_ACCU_OP_SHIFT__*`.

Reply:
Done